src/server/client-route-handler.jsx

```diff
@@ -39,7 +40,9 @@ export default wrap(async (req, res) => {
       latitude: zipInfo.latitude,
       longitude: zipInfo.longitude
     }
-  }).sort((a, b) => new Date(a.date) - new Date(b.date))
+  })
+  .filter((event) => new Date(event.date) >= today)
```

Comment:
Is `date` a string? If so, we should probably not be relying on the default Date constructor since its [wildly inconsistent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) between browsers. I think we have moment as a dependency, which can consistently parse date strings.